commands/release/release.go

```diff
@@ -91,6 +90,67 @@ var ReleaseCmd = &cobra.Command{
 
 var emptyDomain []string = nil
 
+func parseChangeLogFile(changelogFile []byte) (bool, error) {
+	tbdHeaderFound := false
+	isBreakingChange := false
+
+	versionHeaderBool := false
```

Comment:
Needs to be a question (you can drop the `bool` part if you want, though that's just personal preference)